up42/tasking.py

```diff
@@ -105,5 +106,54 @@ def construct_order_parameters(
 
         return order_parameters
 
+    def _query_paginated_output(self, url: str):
+        page = 0
+        response = self.auth._request(request_type="GET", url=url)
+        json_results = response["content"]
+        not_last_page = not response["last"]
+        while not_last_page:
+            page += 1
+            url = replace_page_query(url, page)
+            response = self.auth._request(request_type="GET", url=url)
+            json_results.extend(response["content"])
+            not_last_page = not response["last"]
+        return json_results
+
+    def get_quotations(
+        self,
+        workspace_id: Optional[str] = None,
```

Comment:
ques: The [API documentation ](https://docs.up42.com/developers/api#tag/tasking/operation/listTaskingQuotation)also mention a param `id` (quotation ID).
Maybe we should also add that as well?

Reply:
DONE! Thanks.